addon/mixins/adapter-fetch.js

```diff
@@ -82,7 +82,7 @@ export function determineBodyPromise(response, requestData) {
       }
       const status = response.status;
       if (response.ok && (status === 204 || status === 205 || requestData.method === 'HEAD')) {
```

Comment:
https://github.com/jquery/jquery/blob/2.2-stable/src/ajax.js#L751-L764
jQuery is reading `response.data` if not 204 | 304 | HEAD. Can you verify?

Reply:
Sorry, I don't get what you're asking

Reply:
This doesn't seem to be an issue. We can revisit if it becomes ones